tests/unit/PaymentErrorDialog.spec.ts

```diff
@@ -106,8 +106,39 @@ describe('Payment Error Dialog', () => {
       'We were unable to process your payment due to the following errors:'
     )
     expect(wrapper.findAll('p').at(2).text()).toContain('If this error persists')
-    expect(wrapper.findAll('li').length).toBe(1)
-    expect(wrapper.findAll('li').at(0).text()).toContain(padError[0].message)
+    expect(wrapper.findAll('li').length).toBe(0)
+    expect(wrapper.findAll('span').length).toBe(2)
+    expect(wrapper.findAll('span').at(1).text()).toContain('Your account is in the 3 day PAD confirmation period.')
+
+    expect(wrapper.find(RegistriesContactInfo).exists()).toBe(true)
+    expect(wrapper.find('#dialog-exit-button').exists()).toBe(true)
+
+    wrapper.destroy()
+  })
+
+  it('renders PAD warning messages correctly when they are present', () => {
+    store.state.stateModel.tombstone.authRoles = ['edit', 'view']
+    const wrapper = shallowMount(PaymentErrorDialog,
+      {
+        vuetify,
+        store,
+        propsData: { dialog: true, warnings: [
+          { warning: 'Test Warning 1' },
```

Comment:
Please check the BE code (or ask a BE developer like Argus/Vysakh/Lekshmi) what the message field name is. I'm not convinced that it's "warning".

Reply:
Code has been reverted. If this is found to be incorrect then a future ticket will fix this.